packages/cubejs-query-orchestrator/src/orchestrator/IORedisFactory.ts

```diff
@@ -77,6 +86,14 @@ export async function createIORedisClient(url: string, opts: RedisOptions): Prom
     ...options,
     ...opts,
   });
+
+  client.on('connect', () => logger('IORedis connected'));
```

Comment:
These events can produce a lot of noise inside logs.

Reply:
Only on trace level, right?

Reply:
Right! But the current design of the logger doesn't allow it.